tools/make/tests.toml

```diff
@@ -31,3 +31,23 @@ description = "Run all Rust doctests with all features"
 category = "ICU4X Development"
 command = "cargo"
 args = ["test", "--all-features", "--doc", "--no-fail-fast"]
+
+[tasks.test-cargo-tutorial]
+description = "Build the Cargo tutorial projects"
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+cd docs/tutorials/cargo_tests
+
+pwd = pwd
+set_env CARGO_TARGET_DIR ${pwd}/target
+
+exec --fail-on-error cargo run --release --manifest-path=baked/Cargo.toml
```

Comment:
You can use `-p demo_baked` syntax as it's in-workspace.

Reply:
Done